exonum/src/storage/error.rs

```diff
@@ -30,7 +30,7 @@ impl Error {
     /// ```
     /// use exonum::storage::Error;
     ///
-    /// let error = Error::new("Oh no!");
+    /// let _error = Error::new("Oh no!");
```

Comment:
As for me, it isn't ideal to use underscored variables in the examples. Additionally, "unused variable" warnings should be suppressed in the doc-tests, it is just rust-plugin issue (https://github.com/intellij-rust/intellij-rust/issues/1554).